cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/CoordinatorStreamProperties.java

```diff
@@ -45,6 +47,7 @@ public String toString() {
       .add("notificationThresholdMB", getNotificationThresholdMB())
       .add("generation", getGeneration())
       .add("description", getDescription())
+      .add("ownerPrincipal", getOwnerPrincipal())
```

Comment:
I don't think that's a good idea. Null here represent that this entity (stream) does not have any owner. This is because the user did not provide any owner and then while showing the property back we should retain it as it was given. If we add the hierarchy check here to include the owner name it will be confusing. It's CDAP internal logic to determine whom to run as in absence of an provided owner. Same goes for saying cdap in case of null